src/hooks/core/profile/__mocks__/useProfileHook.ts

```diff
@@ -11,6 +11,7 @@ const useProfileHook = {
   profile: mockSignedProfile,
   signedProfileId,
   anonProfileId,
+  reset: jest.fn(),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `reset` function is a good move for resetting the profile state. However, it's important to ensure that this function is used carefully in other parts of the codebase to avoid unintended side effects. It would be beneficial to add some tests to verify the correct behavior of this function.
```diff
+   reset: jest.fn(),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->